api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker.java

```diff
@@ -40,7 +39,27 @@
                 ),
                 group = KafkaMirrorMaker.RESOURCE_GROUP,
                 scope = KafkaMirrorMaker.SCOPE,
-                version = KafkaMirrorMaker.VERSION
+                version = KafkaConnectS2I.V1BETA1,
+                versions = {
+                        @Crd.Spec.Version(
+                                name = KafkaMirrorMaker.V1BETA1,
+                                served = true,
+                                storage = true
+                        ),
+                        @Crd.Spec.Version(
+                                name = KafkaMirrorMaker.V1ALPHA1,
+                                served = true,
+                                storage = false
+                        )
+                },
+                additionalPrinterColumns = {
```

Comment:
in the case of MM, what do you think about having the bootstrapservers columns for consumer and producer as additional info? Maybe they will take too much space priting on console?

Reply:
Good idea

Reply:
BTW FYI the `priority` on the `@AdditionalPrinterColumn` gives you the ability to include a column only with `-o wide`.

Reply:
So does it mean that you can add consumer and producer bootstrapservers columns to show only if user uses `-o wide` ? Are you going to add this to the PR?